packages/search-ui/src/SearchDriver.js

```diff
@@ -235,13 +250,29 @@ export default class SearchDriver {
         if (this.requestSequencer.isOldRequest(requestId)) return;
         this.requestSequencer.completed(requestId);
 
+        // Results paging start & end
+        const { totalResults } = resultState;
```

Comment:
Do you think we should have this start / end calculation logic covered in a test?

Reply:
Hm, I thought I added them here: 

https://github.com/elastic/search-ui/blob/da835da856df9c86641cd463a6c462803dcbf472/packages/search-ui/src/__tests__/SearchDriver.test.js#L380-L419

Let me know if that covers the calculation logic you'd like to see tested, or if there's anything I'm missing!

Reply:
Oh I missed that, thank you.

Reply:
TBH I wasn't 100% confident where to put the new tests, so happy to reorganize or whatever we prefer :)